data/campaigns/Descent_Into_Darkness/utils/abilities.cfg

```diff
@@ -367,13 +367,15 @@
             animate=no
             kill=no
         [/harm_unit]
-
+        
```

Comment:
These whitespace changes should make the CI check with `wmlindent` fail.

Reply:
Revert whitespace change

Reply:
Revert whitespace change